docs/pages/radiogroup.js

```diff
@@ -0,0 +1,695 @@
+        id="blue"
+        label="Blue"
+        subtext="Typography and shoe in blue"
+        image={<Box height={100} width={80}><Image alt="Typography and shoe in blue" src="https://i.ibb.co/jVR29XV/stock5.jpg" fit="cover" naturalWidth={1} naturalHeight={1}/></Box>}
+        name="Art Preference"
+        onChange={() => setArtPreference( 'blue' )}
+        value="blue"
+      />
+      <RadioGroup.RadioButton
+        checked={artPreference === 'green'}
+        id="green"
+        label="Green"
+        subtext="Abstract art in green"
+        image={<Box height={100} width={80}><Image alt="Abstract art in green" src="https://i.ibb.co/FY2MKr5/stock6.jpg" fit="cover" naturalWidth={1} naturalHeight={1}/></Box>}
+        name="Art Preference"
+        onChange={() => setArtPreference( 'green' )}
+        value="green"
+      />
+    </RadioGroup>
+  );
+}
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="With an error"
+          description="Use `errorMessage` to show an error message below the radio options. Be sure to add `id` as well in order to properly link the message to the RadioGroup."
+        >
+          <MainSection.Card
+            defaultCode={`
+function RadioButtonExample() {
+  const [availability, setAvailability] = React.useState(undefined);
+
+  return (
+    <RadioGroup legend="Which time slot works best for you?" errorMessage="Please select one" id="VariantWithErrorMessage">
+        <RadioGroup.RadioButton
+          checked={availability === 'monday'}
+          id="mondayError"
+          label="Monday"
+          subtext="Morning and afternoon"
+          name="Availability with error"
+          onChange={() => setAvailability( 'monday' )}
+          value="monday"
+        />
+        <RadioGroup.RadioButton
+          checked={availability === 'tuesday'}
+          id="tuesdayError"
+          label="Tuesday"
+          subtext="Morning, afternoon, and evening"
+          name="Availability with error"
+          onChange={() => setAvailability( 'tuesday' )}
+          value="tuesday"
+        />
+        <RadioGroup.RadioButton
+          checked={availability === 'wednesday'}
+          id="WednesdayError"
+          label="Wednesday"
+          subtext="Evening only"
+          name="Availability with error"
+          onChange={() => setAvailability( 'wednesday' )}
+          value="wednesday"
+        />
+    </RadioGroup>
+  );
+}
+          `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Legend visibility"
+          description={`
+            By default, the legend is visible above the items in the RadioGroup. However, if the form items are labeled by content elsewhere on the page, or a more complex legend is needed, the \`legendDisplay\` prop can be used to visually hide the legend. In this case, it is still available to screen reader users, but will not appear visually on the screen.
```

Comment:
"legend" needs backticks throughout, as it's referring to the prop.

Also: is the "?" after "Primary company account goal" expected?